ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -288,6 +289,26 @@ void OwInterface::managePanTilt (const string& opname,
 
   int id = Running.at (opname);
 
+  //if position is over 360 we want to bring it back within the
+  //-360 to 360 range to check if goal position has been reached.
```

Comment:
I haven't examined the math closely but wouldn't it be better to normalize the angle into the range -180 to +180? Anyway if this works for you then no need to fix.

Reply:
I don't have the bandwidth to carefully examine the math here or the similar computations in TestPanTilt.plp.   Something tells me it could be more compact, but I suggest not worry about it if the results appear correct.